system/jlib/jexcept.cpp

```diff
@@ -1354,7 +1360,10 @@ NO_SANITIZE("alignment") void excsighandler(int signum, siginfo_t *info, void *e
         if ( SEHHandler->fireException(new CSEHException(signum,s.str())) )
             return;
     }
-    raise(signum);
+    if ( (SIGABRT == signum) && (processAborted) )
+        _exit(128 + SIGABRT);
```

Comment:
I made it exit exactly like before, just without core.

Reply:
not a big deal either way, as you say the behaviour will be the same.
But there's no intention to use the value (i.e. just treated as a switch), then it would be better if it and the setting function were boolean based.

Reply:
Changed to bool